ui/client/common/ProcessUtils.js

```diff
@@ -155,7 +155,7 @@ class ProcessUtils {
 
   //TODO: this should be done without these switches..
   findNodeDefinitionId = (node) => {
-    switch (node.type) {
+    switch (_.get(node, "type")) {
```

Comment:
I'm wondering if we really need to change this call into lodash method? We do not pass default value, so what are the benefits?

Reply:
Sometimes this method is called with undefined as node. This causes an error described in #511.  
With lodash get there is no need to check if object has type property